libcodechecker/libhandlers/analyze.py

```diff
@@ -177,6 +184,58 @@ def add_arguments_to_parser(parser):
                                help="File containing argument which will be "
                                     "forwarded verbatim for Clang-Tidy.")
 
+    if is_ctu_capable():
+        ctu_opts = parser.add_argument_group('cross translation unit analysis')
+        ctu_mutex_group = ctu_opts.add_mutually_exclusive_group()
+        ctu_mutex_group.add_argument('--ctu',
+                                     action='store_const',
+                                     const=[True, True],
+                                     dest='ctu_phases',
+                                     default=argparse.SUPPRESS,
+                                     help="Perform cross translation unit "
+                                          "(CTU) analysis (both collect and "
+                                          "analyze phases) using default "
+                                          "<ctu-dir> for temporary output. "
+                                          "At the end of the analysis, the "
+                                          "temporary directory is removed.")
```

Comment:
@gerazo Technically it SHOULD NOT. Unless the user's folder structure is special enough that they have enough free space for the reports, but the CTU intermediate files can only be stored on an entire separate path (e.g. a separate network storage or hard drive.)

In that case, I'd tell the user to set `--output-dir` somewhere else, free up space, or use OTF mode.